.travis.yml

```diff
@@ -2,16 +2,25 @@ sudo: required
 dist: trusty
 language: python
 before_install:
-  - node --version
-  - npm install --python=python2 -g dredd
-  - bundle install
+- node --version
+- npm install --python=python2 -g dredd
+- bundle install
 python:
-  - "2.7"
-  - "3.2"
-  - "3.3"
-  - "3.4"
-  - "3.5"
+- '2.7'
+- '3.2'
+- '3.3'
+- '3.4'
+- '3.5'
 script:
-  - python -B -m unittest discover
-  - pip install -e .
-  - bundle exec cucumber
+- python -B -m unittest discover
+- pip install -e .
+- bundle exec cucumber
+deploy:
+  provider: pypi
+  user: wvi
```

Comment:
There is none at the moment, we should create one but I haven't persuaded sre yet.